src/analytics/private/scan-metadata-workflow.ts

```diff
@@ -161,15 +189,44 @@ export class ScanMetadataWorkflow extends Construct {
         level: LogLevel.ALL,
       },
       tracingEnabled: true,
-      comment: 'This state machine is responsible for periodically aggregating events and parameters data and store the aggregated data in DynamoDB',
+      comment: 'This state machine is responsible for aggregating events, parameters and user data and store the aggregated data in DynamoDB',
     });
 
     return scanMetadataStateMachine;
   }
 
+  private checkWorkflowStartFn(props: ScanMetadataWorkflowProps, bucket: IBucket): IFunction {
+    const fnSG = createSGForEgressToAwsService(this, 'CheckWorkflowStartFnSG', props.networkConfig.vpc);
```

Comment:
adopt to #382 that consolidating the security group

Reply:
rebased and modified